.github/workflows/ci.yml

```diff
@@ -66,15 +80,15 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
-      - name: Set up Python ${{ matrix.python-version }}
```

Comment:
Was this working at all? If so, do you know where the matrix was coming from?

Reply:
Based on https://github.com/danpalmer/django-devdata/runs/7772449153?check_suite_focus=true (from the no-op PR #10) it was evaluating to an empty string.